sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -140,19 +140,26 @@ To find the resource ID:
 1. In the resulting JSON, copy the value of the `id` property.
 
 ```C# Snippet:QueryResource
-string resourceId = "/subscriptions/<subscription_id>/resourceGroups/<resource_group_name>/providers/<resource_provider>/<resource>";
 var client = new LogsQueryClient(new DefaultAzureCredential());
 
-Response<LogsQueryResult> result = await client.QueryResourceAsync(
+string resourceId = "/subscriptions/<subscription_id>/resourceGroups/<resource_group_name>/providers/<resource_provider>/<resource>";
+string tableName = "<table_name>";
+Response<LogsQueryResult> results = await client.QueryResourceAsync(
     new ResourceIdentifier(resourceId),
-    "AzureActivity | top 10 by TimeGenerated",
-    new QueryTimeRange(TimeSpan.FromDays(1)));
-
-LogsTable table = result.Value.Table;
+    $"{tableName} | distinct * | project TimeGenerated",
+    new QueryTimeRange(TimeSpan.FromDays(7)));
 
-foreach (LogsTableRow row in table.Rows)
+LogsTable resultTable = results.Value.Table;
+foreach (LogsTableRow rows in resultTable.Rows)
 {
-    Console.WriteLine($"{row["OperationName"]} {row["ResourceGroup"]}");
+    foreach (var row in rows)
```

Comment:
I recommend removing this foreach loop and adding the Console.WriteLine statement back. That way, we show how to access values within the row by name.